projects/storefrontlib/src/lib/cms/components/dynamic-slot/dynamic-slot.component.spec.ts

```diff
@@ -87,34 +90,4 @@ describe('DynamicSlotComponent', () => {
     expect(native.classList.contains('smartEditComponent')).toBeFalsy();
     expect(native.getAttribute('data-smartedit-component-id')).toEqual(null);
   });
-
```

Comment:
Won't we leave one unit test with:
`expect(cmsMappingService.getMappedType).toHaveBeenCalledWith(component)` ?

Reply:
We weren't testing it earlier either (just plain method and those tests were moved/not deleted). Will revisit after refactoring to the directive will be merged (probably before this one).